src/Uno.UI/UI/Xaml/Controls/SwipeControl/SwipeControl.cs

```diff
@@ -0,0 +1,1893 @@
+					//		break;
+					//	case CreatedContent.Top:
+					//		addedVelocity.Y = c_MinimumCloseVelocity;
+					//		break;
+					//	case CreatedContent.Right:
+					//		addedVelocity.X = -c_MinimumCloseVelocity;
+					//		break;
+					//	case CreatedContent.Bottom:
+					//		addedVelocity.Y = -c_MinimumCloseVelocity;
+					//		break;
+					//	case CreatedContent.None:
+					//		break;
+					//	default:
+					//		global::System.Diagnostics.Debug.Assert(false);
+					//		break;
+					//}
+
+					//m_interactionTracker.TryUpdatePositionWithAdditionalVelocity(addedVelocity);
+				}
+			}
+			catch (Exception ex)
+			{
+				Application.Current.RaiseRecoverableUnhandledException(ex);
+			}
+		}
+#endregion
+
+		#region FrameworkElementOverrides
+		protected override void OnApplyTemplate()
+		{
+			ThrowIfHasVerticalAndHorizontalContent(setIsHorizontal: true);
+
+			DetachEventHandlers();
+			GetTemplateParts();
+			EnsureClip();
+			AttachEventHandlers();
+		}
+
+		private void OnPropertyChanged(DependencyPropertyChangedEventArgs args)
+		{
+			DependencyProperty property = args.Property;
+			if (property == LeftItemsProperty)
+			{
+				OnLeftItemsCollectionChanged(args);
+			}
+
+			if (property == RightItemsProperty)
+			{
+				OnRightItemsCollectionChanged(args);
+			}
+
+			if (property == TopItemsProperty)
+			{
+				OnTopItemsCollectionChanged(args);
+			}
+
+			if (property == BottomItemsProperty)
+			{
+				OnBottomItemsCollectionChanged(args);
+			}
+		}
+
+		//Swipe control is usually placed in a list view item. When this is the case the swipe item needs to be the same size as the list view item.
+		//This is to ensure that swiping from anywhere on the list view item causes pointer pressed events in the SwipeControl.  Without this measure
+		//override it is usually not the case that swipe control will fill the available space.  This is because list view item is a content control
+		//and those by convension only provide it's children space for at most their desired size. However list view item itself will take up a different
+		//ammount of space. In the past we solved this issue by requiring the list view item to have the HorizontalContentAlignment and VerticalContentAlignment
+		//set to stretch. This property changes the measure cycle to give as much space as possible to the list view items children.  Instead we can 
+		//just do this ourselves in this measure override and prevent the confusing need for properties set on the parent of swipe control to use it at all.
+		protected override Size MeasureOverride(Size availableSize)
+		{
+			m_rootGrid.Measure(availableSize);
+			Size contentDesiredSize = m_rootGrid.DesiredSize;
+			if (!double.IsInfinity(availableSize.Width))
+			{
+				contentDesiredSize.Width = availableSize.Width;
+			}
+
+			if (!double.IsInfinity(availableSize.Height))
+			{
+				contentDesiredSize.Height = availableSize.Height;
+			}
+
+			return contentDesiredSize;
+		}
+		#endregion
+
+		#region IInteractionTrackerOwner
```

Comment:
Use #if false instead to make comparison with the original code easier.
```suggestion
#if false
		#region IInteractionTrackerOwner
```